R/pkg/R/utils.R

```diff
@@ -697,3 +697,20 @@ is_master_local <- function(master) {
 is_sparkR_shell <- function() {
```

Comment:
also btw, I think we are mixing function name style here - consider naming `isMasterLocal` and `isSparkRShell`